### PR TITLE
Lrate -> LRate. And fix bug in `NeuroModParams.Update()`

### PR DESCRIPTION
**Pull request by garymm**

### CODE REVIEW

File: axon/neuromod.go [hunk 1]
```diff
@@ -85,10 +85,10 @@ const (
 type NeuroModParams struct {
 	DAMod       DAModTypes  `desc:"effects of dopamine modulation on excitatory and inhibitory conductances"`
 	DAModGain   float32     `viewif:"DAMod!=NoDAMod" desc:"multiplicative factor on overall DA modulation specified by DAMod -- resulting overall gain factor is: 1 + DAModGain * DA, where DA is appropriate DA-driven factor"`
-	DALrateMod  slbool.Bool `desc:"modulate learning rate as a function of abs(DA) absolute value of dopamine"`
-	DALratePct  float32     `min:"0" max:"1" viewif:"DALrateMod" desc:"proportion of maximum learning rate that DA can modulate -- e.g., if 0.2, then DA = 0 = 80% of std learning rate, 1 = 100%"`
-	AChLrateMod slbool.Bool `desc:"modulate learning rate as a function of ACh acetylcholine level"`
-	AChLratePct float32     `min:"0" max:"1" viewif:"AChLrateMod" desc:"proportion of maximum learning rate that ACh can modulate -- e.g., if 0.2, then ACh = 0 = 80% of std learning rate, 1 = 100%"`
+	DALRateMod  slbool.Bool `desc:"modulate learning rate as a function of abs(DA) absolute value of dopamine"`
```
Comment (garymm) [inline axon/neuromod.go:88]: from the description, it seems like we don't need these bools; Can't we just set DALRatePct to 0 to disable modulation?

Comment (rcoreilly) [reply axon/neuromod.go:88]: yeah, I often like to have a bool so you can preserve the param value for reference, but in this case, I think it does make sense to just go with the 0 value to turn off.

**Review by rcoreilly — APPROVED** [review]
I will try to remember this LRate thing someday.. :) 

I didn't see any bug fix -- just the rename?

File: axon/neuromod.go [hunk 2]
```diff
@@ -99,17 +99,17 @@ type NeuroModParams struct {
...
+	nm.DALRateMod.SetBool(false)
+	nm.AChLRateMod.SetBool(false)
+	nm.DALRatePct = 0.5
+	nm.AChLRatePct = 0.5
 	nm.BurstGain = 1
 	nm.DipGain = 1
 }
 
 func (nm *NeuroModParams) Update() {
-	mat32.Clamp(nm.DALratePct, 0, 1)
```
Comment (garymm) [inline axon/neuromod.go:111]: This is the bug fix: this code was a no-op before, the return value was being ignored.

File: axon/neuromod.go (hunk 1 above)
Comment (garymm) [reply axon/neuromod.go]: done

**Pull request merged**